crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -570,13 +570,13 @@ fn check_name(checker: &mut Checker, expr: &Expr, range: TextRange) {
         }
 
         // airflow.PY\d{1,2}
-        ["airflow", "PY36"] => Replacement::Name("sys.version_info"),
-        ["airflow", "PY37"] => Replacement::Name("sys.version_info"),
-        ["airflow", "PY38"] => Replacement::Name("sys.version_info"),
-        ["airflow", "PY39"] => Replacement::Name("sys.version_info"),
-        ["airflow", "PY310"] => Replacement::Name("sys.version_info"),
-        ["airflow", "PY311"] => Replacement::Name("sys.version_info"),
-        ["airflow", "PY312"] => Replacement::Name("sys.version_info"),
+        ["airflow", "PY36"] => Replacement::Name("sys.version_info == (3, 6)"),
+        ["airflow", "PY37"] => Replacement::Name("sys.version_info == (3, 7)"),
+        ["airflow", "PY38"] => Replacement::Name("sys.version_info == (3, 8)"),
+        ["airflow", "PY39"] => Replacement::Name("sys.version_info == (3, 9)"),
+        ["airflow", "PY310"] => Replacement::Name("sys.version_info == (3, 10)"),
+        ["airflow", "PY311"] => Replacement::Name("sys.version_info == (3, 11)"),
+        ["airflow", "PY312"] => Replacement::Name("sys.version_info == (3, 12)"),
```

Comment:
Are we sure that's the right recommendation? Can the deprecated Airflow symbols related to this be used with other comparison operators like `> PY36`, `<= PY310`, etc.?

Reply:
ah, you're right. I guess that's why I did not do it this way at the first place. Let me close this one! Thanks for reminding me!